src/extensions/views/EditCustomExtension/EditCustomApp.tsx

```diff
@@ -205,6 +208,11 @@ export const EditCustomExtension: React.FC<OrderListProps> = ({
     return <NotFoundPage backHref={ExtensionsUrls.resolveInstalledExtensionsUrl()} />;
   }
 
+  // Show loading state if data isn't available yet
+  if (loading && !data) {
+    return null; // Or some loading indicator
```

Comment:
can we have one? I assume it's previous implementation, but worth fixing in next PR

Reply:
i forgot to remove this when debugging 🙃